hardhat.config.ts

```diff
@@ -92,16 +92,18 @@ export default {
           apiUrl: 'https://mumbai.polygonscan.com/',
         },
       },
+      live: true,
     },
     polygon: {
       url: process.env.ALCHEMY_POLYGON || '',
-      accounts: [process.env.POLYGON_PRIVATE_KEY || ''],
+      accounts: [process.env.TEST_PRIVATE_KEY || ''],
```

Comment:
The test private key allows us to deploy contracts to our test address instead of the main one that we use for the live version. This is not a permanent change